From 0a0a7bfdd9246e13513766275f3977a7b1640e29 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Fri, 20 Jul 2018 16:08:40 +0800
Subject: [PATCH 4225/5242] MLK-18991-2: arm64:dts:imx8qm-mek-hdmi: change
 HDMI clock source to av pll

commit  f35e46d90a6db9057ab76cac7a2271e86a4edfc5 from
https://source.codeaurora.org/external/imx/linux-imx.git

Change HDMI clock source to av pll.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
(cherry picked from commit e8db637954dd31af85186d6512d749f3b4467317)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts     |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
index 4ef6af4..6d94849 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
@@ -65,9 +65,9 @@
 	assigned-clocks = <&clk IMX8QM_HDMI_PXL_SEL>,
 					<&clk IMX8QM_HDMI_PXL_LINK_SEL>,
 					<&clk IMX8QM_HDMI_PXL_MUX_SEL>;
-	assigned-clock-parents = <&clk IMX8QM_HDMI_AV_PLL_BYPASS_CLK>,
-							<&clk IMX8QM_HDMI_AV_PLL_BYPASS_CLK>,
-							<&clk IMX8QM_HDMI_AV_PLL_BYPASS_CLK>;
+	assigned-clock-parents = <&clk IMX8QM_HDMI_AV_PLL_CLK>,
+							<&clk IMX8QM_HDMI_AV_PLL_CLK>,
+							<&clk IMX8QM_HDMI_AV_PLL_CLK>;
 	fsl,cec;
 	status = "okay";
 };
-- 
1.7.9.5

